### ISSUE TITLE
Triple backtick in gfm-mode (github flavored markdown)

### ISSUE THREAD

**Issue by cowboyd** [Fuco1/smartparens#371]
FYI: I'm happy to try a hand at fixing this if you're willing to guide me.

With github flavored markdown, it is very common to use tripple backtick to demarcate code block. However, with the default smart parens config, typing backtick three times, results in ```` (four) backticks. Is there a way to configure markdown mode to not escape backticks inside backticks

**Comment by Fuco1** [comment]
This is also related to #141.

**Comment by Wilfred** [comment]
Typing a backquote three times in markdown-mode with smartparens gives me:

    ``````

However, typing a backquote three times in gfm-mode with smartparens gives me:

    ```

    ```

    `

**Comment by Fuco1** [comment]
Maybe it has its own "expander" for the backticks?

**Comment by sakshamsharma** [comment]
Long time pending bump?
Sorry, this being in the middle of holidays :)

**Comment by Fuco1** [comment]
I just came across this yesterday updating the readme on one of my projects, he he.

I think this is a `gfm-mode` feature because it also asks me for the "mode" in the minibuffer.  I found the user option `markdown-gfm-use-electric-backquote` which I think controls this.

So the options are to disable that and have SP handle this or disable the tripple pair.

The OPs problem can be fixed by simply declaring tripple backtick as a pair, similarly how in python `"""` is a pair with `"""` (see the python config file).
